hydra-tui/test/Hydra/TUISpec.hs

```diff
@@ -40,61 +40,65 @@ import Hydra.ContestationPeriod (toNominalDiffTime)
 import Hydra.Logging (showLogsOnFailure)
 import Hydra.Network (Host (..))
 import Hydra.Options (ChainConfig (..))
-import Hydra.TUI (runWithVty, tuiContestationPeriod)
+import Hydra.TUI (runWithVty, timeFormatted, tuiContestationPeriod)
 import Hydra.TUI.Options (Options (..))
 import HydraNode (EndToEndLog, HydraClient (HydraClient, hydraNodeId), withHydraNode)
 import System.Posix (OpenMode (WriteOnly), closeFd, defaultFileFlags, openFd)
 
 spec :: Spec
 spec =
-  around setupNodeAndTUI $
-    context "end-to-end smoke tests" $ do
-      it "starts & renders" $
-        \TUITest{sendInputEvent, shouldRender} -> do
-          threadDelay 1
-          shouldRender "TUI"
-          -- Using hex representation of aliceSk's HydraVerificationKey
-          shouldRender "Party 38088e4c2ae82"
-          sendInputEvent $ EvKey (KChar 'q') []
+  describe "Terminal User Interface" $ do
```

Comment:
`XXXSpec.hs` modules do result already in a scope named `XXX`. So we do have a `TUI` context here already, hence this is redundant. 

I think you wanted to separate your unit test from the rest, you could swap lines 51 and 52 for that purpose.

Should: remove this line and swap 51 and 52 to have a context/describe next to your single unit test about time.